aepsych/server/server.py

```diff
@@ -686,8 +717,13 @@ def _configure(self, config):
         self.strat_id = self.n_strats - 1  # 0-index strats
         return self.strat_id
 
-    def configure(self, **config_args):
-        config = Config(**config_args)
+    def configure(self, configasobject=None, **config_args):
```

Comment:
I don't like "configasobject" as a parameter name. Could we just call it "config"?

Reply:
Agreed